### DISCUSSION TITLE
Welcome to UX Tools Discussions!

### DISCUSSION THREAD

**Discussion by luis-valverde-ms** [microsoft/MixedReality-UXTools-Unreal#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kean4711DUB** [comment]
Did openXR plugin replace MR uxtools for ue5?

**Reply by luis-valverde-ms** [reply]
Hi @Kean4711DUB ! OpenXR abstracts devices from different vendors so you can do things like obtaining the motion controller pose in the same way regardless of platform. UXTools works on top of Unreal's abstractions, which in turn use OpenXR, to provide UI controls and interactions that you can use in Mixed Reality. So they are complementary to each other.

**Reply by Kean4711DUB** [reply]
Thank you for the clarification. I know uxtools is not "officially" out on ue5 but they have that section saying it can be used on ue5. Whenever I transfer it, it claims it was built in a different engine. Is there a simple change in the code that will allow it to work on ue5 versus 4.27?

**Reply by luis-valverde-ms** [reply]
We're have a branch with UE5 support working, see issue #46 for details. We're in the process of doing some final testing and will hopefully have an official release within the next couple of weeks.

**Comment by 18084409850** [comment]
ux tool has no sound, and the HoloLens app developed with ue has no sound, what is the reason for this?
